src/query/storage/m3/cluster_resolver_test.go

```diff
@@ -111,3 +114,288 @@ func TestGraphitePath(t *testing.T) {
 		assert.Equal(t, expected[i], cluster.NamespaceID().String())
 	}
 }
+
+func generateClusters(t *testing.T, ctrl *gomock.Controller) Clusters {
+	session := client.NewMockSession(ctrl)
+	retentionFiltered, retentionUnfiltered := time.Hour, time.Hour*10
+	resolution := time.Minute
+
+	clusters, err := NewClusters(UnaggregatedClusterNamespaceDefinition{
+		NamespaceID: ident.StringID("UNAGG"),
```

Comment:
Add the `UNAGG`, `AGG_FILTERED `, etc. as const at the top

Reply:
I've tried this as both constants and strings, and I think the strings actually make the test a bit easier to understand than using constants here